packages/docs/src/demos/fetch/AddonOnNode.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
I suggest adding a pragma comment for Jest to explicitly specify this test running in a Node.js environment:

```js
/**
 * @jest-environment node
 */
```

Without it, JSDOM is assumed as the default environment for this test.

Reply:
as the test is using `@testing-library/react`, wouldn't it use JSDOM anyway?

Reply:
That's the question for us to answer: is the support target Node.js or JSDOM? Those are quite different environments, and we can use MSW in both of them. It'd mainly define the test setup we have to ensure this support. 

So it looks like you're aiming at supporting JSDOM, correct?

Reply:
I'm not sure if I follow 100%, but when a developer uses this addon, it's in the context of Storybook, which relates to rendering of components. Either that will happen in the browser when using Storybook, or in JSDOM when using jest + any rendering engine (enzyme, rtl, etc). So I suppose yes, JSDOM is the main target. I don't know if there's a scenario where people would use the server from this addon rather than using the one from msw directly